apps/ledger-live-desktop/src/renderer/families/polkadot/operationDetails.jsx

```diff
@@ -29,13 +29,13 @@ import { useDiscreetMode } from "~/renderer/components/Discreet";
 import { urls } from "~/config/urls";
 import { SplitAddress } from "~/renderer/components/OperationsList/AddressCell";
 
-function getURLFeesInfo(op: Operation): ?string {
+function getURLFeesInfo(op: Operation, currencyId: string): ?string {
```

Comment:
I might be missing something but currencyId is not used here right ?

Reply:
I have added a parameter in /ledger-live/apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.jsx
```
  const urlFeesInfo =
    specific &&
    specific.getURLFeesInfo &&
    specific.getURLFeesInfo(operation, mainAccount.currency.id);
```

I need to distinguish different currency in cosmos family.
As this method is shared by other families, this is why I need to change the signature for other families